{"name": "反向用例: raise TypeError", "status": "passed", "steps": [{"name": "调用相加", "status": "broken", "statusDetails": {"message": "TypeError: unsupported operand type(s) for +: 'int' and 'list'\n", "trace": "  File \"/Users/linhao/PycharmProjects/pythonProject/homework/venv_new/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/Users/linhao/PycharmProjects/pythonProject/homework/Test_2021_06_17/calculator/calculator.py\", line 7, in add\n    return a + b\n"}, "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "[1, 2]"}], "start": 1624197780305, "stop": 1624197780305}], "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "[1, 2]"}, {"name": "error_type", "value": "'TypeError'"}], "start": 1624197780305, "stop": 1624197780305, "uuid": "72652bba-099f-4d01-a7e0-ae6311aeea38", "historyId": "07925969ec676fc6cd03bb705509bfe0", "testCaseId": "1b5aab2181c21923f6bd8223ee598e23", "fullName": "Test_2021_06_17.test_cal.test_cal.TestAddDivision#test_add_fail", "labels": [{"name": "story", "value": "测试加法"}, {"name": "feature", "value": "测试计算器"}, {"name": "parentSuite", "value": "Test_2021_06_17.test_cal"}, {"name": "suite", "value": "test_cal"}, {"name": "subSuite", "value": "TestAddDivision"}, {"name": "host", "value": "192.168.8.102"}, {"name": "thread", "value": "58762-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_2021_06_17.test_cal.test_cal"}]}